### DISCUSSION TITLE
Moving external images out side of the CRDs

### DISCUSSION THREAD

**Discussion by arpan57** [hyperspike/valkey-operator#45] [General]
Hi @dmolik , 

Following two external images should be ideally externalised and configurable and referred to in the CRD file - config/chart/crds/valkey-crd.yaml
line 79 and 83     

           default: docker.io/bitnami/redis-exporter:1.62.0-debian-12-r2
                description: Exporter Image to use
                type: string
              image:
                default: docker.io/bitnami/valkey-cluster:7.2.6-debian-12-r0
                description: Image to use

Regards,
Arpan

**Comment by dmolik** [comment]
that's fair, shouldn't be too hard to implement.

**Comment by dmolik** [comment]
newest release should be good to go:

I've updated the release process:
```sh
LATEST=$(curl -s https://api.github.com/repos/hyperspike/valkey-operator/releases/latest | jq -cr .tag_name)
curl -sL https://github.com/hyperspike/valkey-operator/releases/download/$LATEST/install.yaml | kubectl create -f -
```
